substrate/bin/utils/chain-spec-builder/src/lib.rs

```diff
@@ -171,153 +170,32 @@ struct DefaultCmd {
 /// can be updated with the runtime provided in the command line.
 #[derive(Parser, Debug, Clone)]
 pub struct EditCmd {
-	/// Chain spec to be edited
+	/// Chain spec to be edited.
 	#[arg(long, short)]
 	pub input_chain_spec: PathBuf,
-	/// The path to new runtime wasm blob to be stored into chain-spec
+	/// The path to new runtime wasm blob to be stored into chain-spec.
 	#[arg(long, short = 'r')]
 	pub runtime_wasm_path: Option<PathBuf>,
-	/// Convert genesis spec to raw format
+	/// Convert genesis spec to raw format.
 	#[arg(long, short = 's')]
 	pub convert_to_raw: bool,
 }
 
-/// Verifies provided input chain spec. If the runtime is provided verification is performed against
+/// Verifies provided input chain spec.
 /// new runtime.
 #[derive(Parser, Debug, Clone)]
 pub struct VerifyCmd {
-	/// Chain spec to be edited
+	/// Chain spec to be edited.
 	#[arg(long, short)]
```

Comment:
```suggestion
```
What do you think about removing it? Then we could to `chain-spec-builder verify chain_spec.json` without the need to always specify the long commands. I think there are also some other commands that take only one command but still require knowledge of a command line flag. But its only a nit, up to you.

Reply:
make sense, fixed.